components/progressbar/index.css

```diff
@@ -223,7 +223,7 @@
 }
 
 /* Static White */
-.spectrum-ProgressBar--staticWhite {
+.spectrum-ProgressBar.spectrum-ProgressBar--staticWhite {
```

Comment:
I don't love that we have to increase specificity to make this work but we can address those concerns in a separate PR because it's an issue we have across the project.